src/encoded/types/dataset.py

```diff
@@ -363,12 +363,10 @@ def superseded_by(self, request, superseded_by):
         'title': "Publication file set",
         'description': 'A set of files that are described/analyzed in a publication.',
     })
-class PublicationData(FileSet, CalculatedFileSetBiosample, CalculatedFileSetAssay, CalculatedAssaySynonyms):
+class PublicationData(FileSet):
     item_type = 'publication_data'
     schema = load_schema('encoded:schemas/publication_data.json')
     embedded = [
```

Comment:
Do we need a comment that this DS does NOT inherit embedded from FileSet/Dataset?